april_vision/cli/marker_generator/marker_modes/mode_single.py

```diff
@@ -20,7 +22,13 @@ def main(args: argparse.Namespace) -> None:
 
     marker_ids = parse_marker_ranges(tag_data, args.range)
 
-    page_size = PageSize[args.page_size]
+    page_size: Union[PageSize, CustomPageSize]
+    if args.page_size == 'CROPPED':
+        # Allow for an additional marker pixel border
+        required_width = args.marker_size * (12 / 8)
```

Comment:
This looks like it is hard coded for 25h11, it should be made generic to support all marker types